client/app/src/accounts/account.controller.js

```diff
@@ -1523,6 +1528,93 @@
         }
       };
 
+      function upload() {
+        var options = {
+          title: "Upload Image",
+          filters: [
+            { name: 'Images', extensions: ['jpg', 'png', 'gif'] }
+          ],
+          properties: ["openFile"],
+        };
+        var userPath = 'assets/images/user/'
+        var dirPath = path.resolve(__dirname, userPath);
```

Comment:
Semicolon consistency (Need that linter ASAP @j-a-m-l )

Reply:
Added the semicolon. That was my bad. Thanks!